### PR TITLE
General links and corrections

### PR DESCRIPTION
**Pull request by williamsmithza**
Just a couple of suggested updates to the descriptions and links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,11 +21,11 @@ https://discord.gg/4mVaTsH
 
 #### Requirements
 * Coin daemon(s) (find the coin's repo and build latest version from source)
-* [Node.js](http://nodejs.org/) v7+ ([follow these installation instructions](https://github.com/joyent/node/wiki/Installing-Node.js-via-package-manager))
+* [Node.js](http://nodejs.org/) v7+ ([follow these installation instructions](https://github.com/nodejs/node))
```
Comment (egyptianbman) [inline README.md]: Can you replace `v7+` with `v8.11`?

**Review by egyptianbman — CHANGES_REQUESTED** [review]
One minor request -- otherwise, looks great; thanks!

File: README.md (hunk 1 above)
Comment (williamsmithza) [reply README.md]: Thanx

**Review by egyptianbman — APPROVED** [review]
Thanks!

**Pull request merged**